tests/fsharpqa/Source/InteractiveSession/Paket/PaketWithBrokenDepsFile/plot.fsx

```diff
@@ -1,3 +1,5 @@
+//<Expects status="Error" span="(5,6)" id="FS0039">The namespace or module 'XPlot' is not defined.</Expects>
```

Comment:
yes they are currently not reported because: https://github.com/Microsoft/visualfsharp/issues/2489